docker-compose.prod.yml

```diff
@@ -39,9 +39,7 @@ services:
       - "443:443"
 
   django:
-    build:
-      context: .
-      dockerfile: ./dev/django.Dockerfile
```

Comment:
let's leave the build context until we get the actions building images properly and publicly available .  After we can update the deployment instructions.

Reply:
Re-added in 86616ee